persistence/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/TestEntity.scala

```diff
@@ -43,9 +43,9 @@ object TestEntity {
     override def aggregateTag: AggregateEventShards[Evt] = Evt.aggregateEventShards
   }
 
-  final case class Appended(element: String) extends Evt
+  final case class Appended(entityId: String, element: String) extends Evt
```

Comment:
@johanandren I think you can take a stab at this. Search for `eventsByTag`. There you see that we emit tuple of Event and Offset. Instead we can define an envelope class that contains Event, Offset and entityId.

You grab the entityId from the `persistenceId` from the `UUIDEventEnvelope` in the underlying query.